src/main/server/enable_ssl.sh

```diff
@@ -20,15 +20,16 @@ do
     d) domain=${OPTARG};;
     v) dns_validate=${OPTARG};;
     r) renew_flag='true';;
+    e) email=${OPTARG};;
    esac
 done
 
 ERROR_MESSAGE="There is a problem in installing SSL to Ant Media Server.\n Please take a look at the logs above and try to fix.\n If you do not have any idea, contact@antmedia.io"
 
 usage() {
   echo "Usage:"
-  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}]"
-  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}] [-v {route53 or custom}]"
+  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}] -e {YOUR_EMAIL}"
```

Comment:
It seems that e-mail is optional. AFAIK, optional parameters are shown with square brackets